Admin/AbstractAdmin.php

```diff
@@ -422,8 +422,6 @@
      */
     protected $securityInformation = array();
 
-    protected $cacheIsGranted = array();
```

Comment:
Removing a protected method is a BC-break. Accessing it could be deprecated mith magic methods if deemed worth it. WDYT @sonata-project/contributors ?